### PR TITLE
8291828: Reduce runtime of java.util.stream microbenchmarks

### PR DESCRIPTION
**Pull request by ericcaspole**
These changes reduce the default run time from about 1 day 22 hours to about 6 hours with good stability we can use in weekly testing.

---------
### Progress
- [x] Change must be properly reviewed (1 review required, with at least 1 [Reviewer](https://openjdk.org/bylaws#reviewer))
- [x] Change must not contain extraneous whitespace
- [x] Commit message must refer to an issue

### Issue
 * [JDK-8291828](https://bugs.openjdk.org/browse/JDK-8291828): Reduce runtime of java.util.stream microbenchmarks

### Reviewers
 * [Roger Riggs](https://openjdk.org/census#rriggs) (@RogerRiggs - **Reviewer**)

### Reviewing
<details><summary>Using <code>git</code></summary>

Checkout this PR locally: \
`$ git fetch https://git.openjdk.org/jdk pull/9733/head:pull/9733` \
`$ git checkout pull/9733`

Update a local copy of the PR: \
`$ git checkout pull/9733` \
`$ git pull https://git.openjdk.org/jdk pull/9733/head`

</details>
<details><summary>Using Skara CLI tools</summary>

Checkout this PR locally: \
`$ git pr checkout 9733`

View PR using the GUI difftool: \
`$ git pr show -t 9733`

</details>
<details><summary>Using diff file</summary>

Download this PR as a diff file: \
<a href="https://git.openjdk.org/jdk/pull/9733.diff">https://git.openjdk.org/jdk/pull/9733.diff</a>

</details>

### CODE REVIEW

File: test/micro/org/openjdk/bench/java/util/stream/ops/value/SizedCount.java [hunk 1]
```diff
@@ -26,6 +26,8 @@
 import org.openjdk.jmh.annotations.BenchmarkMode;
 import org.openjdk.jmh.annotations.Fork;
 import org.openjdk.jmh.annotations.Measurement;
+import org.openjdk.jmh.annotations.Fork;
+import org.openjdk.jmh.annotations.Measurement;
```
Comment (RogerRiggs) [inline test/micro/org/openjdk/bench/java/util/stream/ops/value/SizedCount.java:30]: Should there have been annotations for Fork to go with this import?
Importing Measurement again is a dup.

File: test/micro/org/openjdk/bench/java/util/stream/ops/value/SizedSum.java [hunk 2]
```diff
@@ -26,6 +26,8 @@
 import org.openjdk.jmh.annotations.BenchmarkMode;
 import org.openjdk.jmh.annotations.Fork;
 import org.openjdk.jmh.annotations.Measurement;
+import org.openjdk.jmh.annotations.Fork;
+import org.openjdk.jmh.annotations.Measurement;
 import org.openjdk.jmh.annotations.Mode;
 import org.openjdk.jmh.annotations.OutputTimeUnit;
 import org.openjdk.jmh.annotations.Param;
```
Comment (RogerRiggs) [inline test/micro/org/openjdk/bench/java/util/stream/ops/value/SizedSum.java:33]: Ditto imports vs annotations.

File: test/micro/org/openjdk/bench/java/util/stream/ops/value/SizedCount.java (hunk 1 above)
Comment (ericcaspole) [reply test/micro/org/openjdk/bench/java/util/stream/ops/value/SizedCount.java:30]: Good eye, this one has some mistake, I will fix these and get a new time estimate overnight.

**Comment by ericcaspole** [comment]
These last changes reduce the run time to about 4h45m with adequate result stability.

File: test/micro/org/openjdk/bench/java/util/stream/ops/value/SizedSum.java (hunk 2 above)
Comment (ericcaspole) [reply test/micro/org/openjdk/bench/java/util/stream/ops/value/SizedSum.java]: These last changes reduce the run time to about 4h45m with adequate result stability.

**Review by RogerRiggs — APPROVED** [review]

**Comment by ericcaspole** [comment]
Thanks Roger!

**Pull request closed without merging**